docs-chef-io/content/inspec/resources/file.md

```diff
@@ -76,134 +80,233 @@ where
+The `file_version` property tests if a Windows file's version matches the specified value. The difference between a file's, 'file version' and 'product version' is that the file version is the version number of the file itself, whereas the product version is the version number associated with the application from which that file originates:
 
+```ruby
     its('file_version') { should eq '1.2.3' }
+```
 
 ### group
 
 The `group` property tests if the group to which a file belongs matches the specified value.
 
+```ruby
     its('group') { should eq 'admins' }
+```
 
 The following examples show how to use this Chef InSpec audit resource.
 
 ### link_path
 
-The `link_path` property tests if the file exists at the specified path. If the file is a symlink,
-Chef InSpec will resolve the symlink recursively and return the ultimate linked file.
+The `link_path` property tests if the file exists at the specified path. If the file is a symlink, the Chef InSpec resolves the symlink recursively and return the ultimate linked file.
 
+```ruby
     its('link_path') { should eq '/some/path/to/file' }
+```
 
 ### shallow_link_path
 
-The `shallow_link_path` property returns the path that the file refers to, only resolving
-it once (that is, it performs a readlink operation). If the file is not a symlink, nil is returned.
+The `shallow_link_path` property returns the path that the file refers to, only resolving it once (that is, it performs a readlink operation). If the file is not a symlink, **nil** is returned.
 
+```ruby
     its('shallow_link_path') { should eq '/some/path/to/file' }
+```
 
 ### md5sum
 
 The `md5sum` property tests if the MD5 checksum for a file matches the specified value.
 
+```ruby
     its('md5sum') { should eq '3329x3hf9130gjs9jlasf2305mx91s4j' }
+```
 
 ### mode
 
 The `mode` property tests if the mode assigned to the file matches the specified value.
 
+```ruby
     its('mode') { should cmp '0644' }
+```
 
-Chef InSpec [octal](https://en.wikipedia.org/wiki/Leading_zero#0_as_a_prefix) values begin the numeric mode specification with zero.
+The Chef InSpec [octal](https://en.wikipedia.org/wiki/Leading_zero#0_as_a_prefix) values begin the numeric mode specification with zero.
 
 For example, write:
 
+```ruby
     { should cmp '0644' }
+```
 
 not
 
+```ruby
     { should cmp '644' }
+```
 
 or write:
 
+```ruby
     { should cmp '01775' }
+```
 
 not
 
+```ruby
     { should cmp '1775' }
+```
 
-Without the zero prefix for the octal value, Chef InSpec will interpret it as the _decimal_ value 644, which is octal 1024 or `-----w-r-T`, and any test for a file that is `-rw-r--r--` will fail.
+Without the zero prefix for the octal value, the Chef InSpec interprets it as the _decimal_ value 644, which is octal 1024 or `-----w-r-T`, and any test for a file that is `-rw-r--r--` will fail.
 
-Note: see the [`be_more_permissive_than(mode)`](<#be_more_permissive_than?(mode)>) matcher for upper and lower bounds on file mode.
+{{< note >}}
```

Comment:
It seems the note tag is not rendered.
<img width="728" alt="image" src="https://user-images.githubusercontent.com/98935583/163817846-f0313cad-c172-4366-8526-cab06a57eaa6.png">